### PR TITLE
[DOCFIX] Combine TTL into storage management and add todos

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/advanced/Alluxio-Storage-Management.md [hunk 1]
```diff
@@ -233,17 +173,183 @@ specific alias. For example, at most 1 tier can have the alias HDD. If you want
...
+you can also develop your own evictor appropriate for your workload.
+
+When using synchronous eviction, it is recommended to use smaller block sizes (around 64-128MB),
+to reduce the latency of block eviction. When using the space reserver, block size does not affect
+eviction latency.
+
+## Operate Data in Alluxio
+
+TODO(Zac): Add documentation on check capacity, fraction of used capacity and etc
+TODO(Zac): Add documentation for free, load, and persist commands.
```
Comment (gpang) [inline docs/en/advanced/Alluxio-Storage-Management.md]: TTL seems like it should be in this list?

Also, how are these sections different from the sections in the CLI documentation?

If someone asks a question "how do I use the [TTL|free|persist] feature", which page do we refer them to?

File: docs/en/advanced/Alluxio-Storage-Management.md [hunk 2]
```diff
@@ -233,17 +173,183 @@ specific alias. For example, at most 1 tier can have the alias HDD. If you want
...
 
-These are the configuration parameters for tiered storage.
+```
+alluxio.worker.tieredstore.level0.watermark.high.ratio=0.9 # 216GB * 0.9 ~ 200GB
+alluxio.worker.tieredstore.level0.watermark.low.ratio=0.75 # 216GB * 0.75 ~ 160GB
+```
+
+In write or read-cache heavy workloads, asynchronous eviction can improve performance.
+
+Synchronous eviction waits for a client to request more space than is currently available on the worker
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/advanced/Alluxio-Storage-Management.md]: I think we should still document it as it is still supported and I know deployments using this feature. But we should make it explicit that this is deprecated by the async version.

File: docs/en/advanced/Alluxio-Storage-Management.md [hunk 3]
```diff
@@ -233,17 +173,183 @@ specific alias. For example, at most 1 tier can have the alias HDD. If you want
...
+alluxio.worker.tieredstore.level0.watermark.low.ratio=0.75 # 216GB * 0.75 ~ 160GB
+```
+
+In write or read-cache heavy workloads, asynchronous eviction can improve performance.
+
+Synchronous eviction waits for a client to request more space than is currently available on the worker 
+and then kicks off the eviction process to free up enough space to serve that request. This leads to many 
+small eviction attempts, which is less efficient but maximizes the utilization of available Alluxio space.
+
+### Evictors
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/advanced/Alluxio-Storage-Management.md]: SG. added into a TODO

File: docs/en/advanced/Alluxio-Storage-Management.md (hunk 1 above)
Comment (apc999) [reply docs/en/advanced/Alluxio-Storage-Management.md]: In here I am thinking a high-level overview, maybe through a running example to show how to manage data in Alluxio with the  simplest use of these CLIs. The CLI doc is more comprehensive on the detailed syntax.

File: docs/en/advanced/Alluxio-Storage-Management.md [hunk 4]
```diff
@@ -233,17 +173,183 @@ specific alias. For example, at most 1 tier can have the alias HDD. If you want
...
+    only evict from that StorageDir.
+
+In the future, additional evictors will be available. Since Alluxio supports custom evictors,
+you can also develop your own evictor appropriate for your workload.
+
+When using synchronous eviction, it is recommended to use smaller block sizes (around 64-128MB),
+to reduce the latency of block eviction. When using the space reserver, block size does not affect
+eviction latency.
+
+## Operate Data in Alluxio
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/advanced/Alluxio-Storage-Management.md]: Put TTL as a subsection of this section

**Comment by apc999** [comment]
@gpang @aaudiber  PTAL

File: docs/en/advanced/Alluxio-Storage-Management.md (hunk 1 above)
Comment (gpang) [reply docs/en/advanced/Alluxio-Storage-Management.md]: @apc999 
That means this page should include all the ways you can "manage data" in Alluxio, and explain what it means and the implications, right? For example, to free a file, this page should discuss the "free" concept, and then present how to free a file. The CLI command for free is only 1 way to do that, and there is a java api for it as well.

Shouldn't this model be followed for all possible "data management" operations like TTL, free, persist, etc?

Comment (apc999) [reply docs/en/advanced/Alluxio-Storage-Management.md]: @gpang  as we discussed, we will move the Java-level examples.

**Pull request merged**